crates/cxx-qt-gen/src/syntax/foreignmod.rs

```diff
@@ -37,7 +37,7 @@ pub(crate) fn foreign_mod_to_foreign_item_types(
         .filter_map(|item| match foreign_item_to_type(item) {
             Ok(Some(value)) => Some(Ok(value)),
             Ok(None) => None,
-            Err(err) => Some(Err(err)),
+            Err(err) => Some(Err(err)), // Hard to cover since foreign_item_to_type never returns error
```

Comment:
can remove this comment as we have an error case now?

Reply:
done